README.Rmd

```diff
@@ -67,27 +67,30 @@ Because our office (mostly) cannot observe individual condo unit characteristics
 library(dplyr)
 library(tidyr)
 library(yaml)
-params <- read_yaml("params.yaml")
+library(httr)
+
+condo_params <- read_yaml("params.yaml")
+condo_preds <- condo_params$model$predictor$all
+
+res_params_text <- GET(
+  "https://raw.githubusercontent.com/ccao-data/model-res-avm/master/params.yaml"
+) %>%
+  content(as = "text")
```

Comment:
**nitpick:** No need to do a GET or load `httr`, `read_yaml()` can ingest directly from a URL.

Reply:
Ooh that's nice! Simplified in [6eec8f6](https://github.com/ccao-data/model-condo-avm/pull/4/commits/6eec8f67b974531ff617353daef37dff54f541ce) 👍🏻